test/fail_compilation/failcstuff1.c

```diff
@@ -1,7 +1,6 @@
 // check the expression parser
 /* TEST_OUTPUT
 ---
-fail_compilation/failcstuff1.c(51): Error: attributes should be specified before the function definition
```

Comment:
Can you just move this test to test2266.c instead?

Reply:
I just removed test22666.c instead.

Reply:
I think the point was that the test is now compilable and therefore it should be in the compilable directory. Right now you are putting a compilable test case in a fail_compilation file.